app/models/invoice_item.rb

```diff
@@ -0,0 +1,12 @@
+class InvoiceItem < ApplicationRecord
+  belongs_to :item
+  belongs_to :invoice
+  has_many :merchants, through: :items
+  has_many :transactions, through: :invoices
+
+  validates :quantity, presence: true
+  validates :unit_price, presence: true
+
+  enum status: {:pending => 0, :packaged => 1, :shipped => 2} #used in the InvoiceItems Controller, Status action
```

Comment:
Thank you for sharing info about this! Good to know! We'll update the migrations and re-do the schema to account for the change in datatype.